src/Raven.Client/Documents/Identity/HiLoResult.cs

```diff
@@ -13,9 +13,7 @@ namespace Raven.Client.Documents.Identity
     /// </summary>   
     public class HiLoResult
     {
-        /// <summary>
-        /// prefix string, including the server prefix (if exists)
-        /// </summary>
+
         public string Prefix { get; set; }
```

Comment:
I did it at first without looking too much into what it is. Apparently the prefix is the collection name.